Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs

```diff
@@ -103,11 +102,34 @@ public void StartServiceAfterUnhandledException()
 
             // Some exception occurred.
             ApplicationLifecycleHelper.Instance.InvokeUnhandledExceptionOccurred(null, new Exception());
+            Task.Delay(100).Wait();
 
             // Start any service.
             AppCenter.Start(typeof(TestAppCenterService));
 
-            // No exception throws, just log message.
+            // No exceptions are thrown.
+            // TODO: Verify that there are no errors in logs (System.Diagnostics.Debug.Listeners is not available here).
```

Comment:
I think we will not address that todo, please remove. Maybe keep comment that we cannot verify this but no TODO keyword.